### ISSUE TITLE
Feature Request: Shared Activities

### ISSUE THREAD

**Issue by sehaubold** [crash1115/5e-training#56]
**TLDR**: Link progress of multiple entries using IDs from the read-only field or some other means

I have loved the module so far, its great for tracking different individual activities. Recently the party has gotten to an area where they will be working as a group to research a grimoire. I have been manually adjusting the values in their books so far, but it would be great if I could include the item ID in a semicolon separated field to link the progress bar. Potentially one sheet is the master entry or a separate screen that you populate with group tasks and a list of the relevant characters?

**Comment by crash1115** [comment]
Hey, thanks for the suggestion. There's a planned feature for somewhere down the line to implement what I call "Global Activities", which pretty much fill the use case you described. GM can configure items that display on all PC sheets, or all NPC sheets, or, potentially, somewhere on the canvas (think, countdown timer, or quest tracker ui from an MMO).

In the meantime, you can implement this sort of "progress link" by using macro activities in conjunction with the Advanced Macros module (used to be part of The Furnace). If you wanna shoot me a message on Discord, I can help you write the macros to suit your needs.

**Comment by sehaubold** [comment]
Ooooo! Sweet! Alright I may take you up on that then! Let me look into the
specifics of those macros again so I'm prepped when I message you lol
